packages/templating-tools/package.js

```diff
@@ -16,7 +16,7 @@ Package.onUse(function(api) {
     // boilerplate-generator uses spacebars-compiler.)
     // XXX maybe uglify should be applied by this plugin instead of via magic
     // weak dependency.
-    'minifier-js@1.2.14'
+    'minifier-js@1.2.14 || 2.0.0'
```

Comment:
Ok, no errors on the meteor side, except `blaze` (still) and Mongo oplog, which sometimes passes sometimes fails (I think that one is related to my PC), and DDP which got a timeout instead of a rate-limit, which again I think is my PC. `spacebars-compiler` and `templating-tools` both pass, though, on client and server, which means to me that either `minifier-js` is not needed by `templating-tools` or it's not tested. Another thing of note is that *everything* sent to the client is concatenated and minified by default, courtesy of `isobuild`, so long as you aren't in development mode, so test and production modes will be minified by default.